dataplane/processor.go

```diff
@@ -229,7 +229,10 @@ func (p *Processor) UpdateConfig(resource, sampler string, config *control.Sampl
 			}
 		}
 		if eventor != nil {
-			eventor.SetEventsConfig(config.Events)
+			err := eventor.SetEventsConfig(config.Events)
```

Comment:
we should configure a static check in the github pipeline that alerts us of unhandled errors

Reply:
Done!